File: pset6/mario/mario.py
import cs50

ask_again = True

while ask_again:
    num_levels = cs50.get_int("How many levels do you want in your pyramid. Enter an integer between 1 and 8 inclusive ")
    ask_again = (not(num_levels >= 1 and num_levels <= 8))

for i in range(1, num_levels + 1):
    print((" " * (num_levels - i)) + ("#" * i) + "  " + ("#" * i))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter the number of levels you want to have: ");
    }
    while (!(height >= 1 && height <= 8));
    for (int i = 0, numSpaces = height - 1, numHashtags = 1; i < height; i++)
    {
        for (int j = 0; j < numSpaces; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < numHashtags; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < numHashtags; j++)
        {
            printf("#");
        }
        printf("\n");
        numSpaces--;
        numHashtags++;
    }
}

